pwndbg/arguments.py

```diff
@@ -201,5 +201,11 @@ def format_args(instruction):
     for arg, value in get(instruction):
         code   = arg.type != 'char'
         pretty = pwndbg.chain.format(value, code=code)
+
+        # Enhance args display
+        if arg.name == 'fd' and isinstance(value, int):
+            path = pwndbg.file.readlink('/proc/%d/fd/%d' % (pwndbg.proc.pid, value))
```

Comment:
Does this fail gracefully for qemu-user?

Reply:
That's why its not merged yet. I haven't checked that :P.